main.go

```diff
@@ -22,7 +22,13 @@ type Params struct {
 	Username      *string  `json:"username,omitempty"`
 }
 
+var (
+	buildDate string
```

Comment:
This is already integrated into most plugins. The makefile have been discussed with brad.